gissupport_plugin/tools/widgets/gs_select_area.py

```diff
@@ -24,6 +24,8 @@ class GsSelectAreaOption(Enum):
 
 class GsSelectArea(QWidget, FORM_CLASS):
     methodChanged = pyqtSignal()
+    # NOWY SYGNAŁ: Przekazuje gotową geometrię do downloadera
```

Comment:
Usuń tekst `NOWY SYGNAŁ`. Teraz może jest nowy, ale to zostaje i cały czas będzie pisać, że nowy, nawet jak będzie stary. Ja widzę, że jest nowy dzięki PR